packages/server/src/routes/users.js

```diff
@@ -60,6 +60,19 @@ router.post('/', requireAdminUser, async (req, res, next) => {
     }
 });
 
+router.put('/:userId', requireUser, async (req, res) => {
+    const user = req.body;
```

Comment:
Should I be doing any checks here?

Reply:
Per my broader feedback, if this route was changed to a `PATCH`, I think it would be good to validate that only a `name` attribute is provided in the request body. 

Something that amounts to the following would be great:
```js
const allowedFields = new Set([ 'name' ]);
for (cons key of Object.keys(req.body)) {
    if (!allowedFields.has(key)) {
        res.status(400).json({ message: `Request body contains unsupported field: ${key}` });
        return;
    }
}

const user = req.body;  // Note: no destructuring, as we expect req.body to represent a user object
try {  // proceed as-written...
    const result = await db.updateUser(user); 
```